dispatchers.go

```diff
@@ -179,13 +187,19 @@ func dispatchCopy(b *Builder, args []string, attributes map[string]bool, flagArg
 		switch {
 		case strings.HasPrefix(arg, "--chown="):
 			chown = strings.TrimPrefix(arg, "--chown=")
+		case strings.HasPrefix(arg, "--chmod="):
+			chmod = strings.TrimPrefix(arg, "--chmod=")
+			_, err = strconv.ParseUint(chmod, 8, 32)
+			if err != nil {
+				return errChmodConversion(chmod)
+			}
```

Comment:
nit, you might consider renaming errChmodConversion to something like checkChmodConversion then putting the ParseUnit() call into and then return err or nil

Reply:
This is exactly what I had (even the same name), but then spotted err bla bla... funtions at the bottom and decided to follow this convention. Anyway, I will fix it right away.